Misc/NEWS.d/next/Windows/2022-06-20-22-32-14.gh-issue-94018.bycC3A.rst

```diff
@@ -0,0 +1 @@
+Remove trailing spaces in :func:`zipfile._sanitize_windows_name`
```

Comment:
`_sanitize_windows_name` is not a public or documented function, so it should not be mentioned here.

```suggestion
:mod:`zipfile` will now remove trailing spaces from path components on extraction.
```